modules/library/cql/src/test/java/org/geotools/filter/text/ecql/ECQLComparisonPredicateTest.java

```diff
@@ -57,6 +57,19 @@ public ECQLComparisonPredicateTest() {
         super(Language.ECQL);
     }
 
+    /**
+     * Equals predicate sample
+     *
+     * @throws Exception
+     */
+    @Test
+    public void deprecatedPredicate() throws Exception {
```

Comment:
Should test all the added operators  and verify for reach that the result is the expected one

Reply:
Ok. I was just using the same tests available on the CQLComparisonPredicateTest.
I'm going to add new tests then.

Reply:
@aaime Tests added